Packs/AlibabaActionTrail/Playbooks/Alibaba_ActionTrail_-_multiple_unauthorized_action_attempts_detected_by_a_user.yml

```diff
@@ -0,0 +1,635 @@
+    scriptarguments:
+      AutoCommit:
+        simple: "No"
+      CustomBlockRule:
+        simple: "True"
+      IP:
+        complex:
+          root: alert
+          accessor: remoteip
+          transformers:
+          - operator: uniq
+      InputEnrichment:
+        simple: "False"
+      InternalRange:
+        complex:
+          root: inputs.InternalRange
+      RuleDirection:
+        simple: inbound
+      RuleName:
+        simple: XSOAR - Block IP playbook - ${alert.alertid}
+      UserVerification:
+        simple: "False"
+    separatecontext: true
+    continueonerrortype: ""
+    loop:
+      iscommand: false
+      exitCondition: ""
+      wait: 1
+      max: 100
+    view: |-
+      {
+        "position": {
+          "x": 90,
+          "y": 1280
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "35":
+    id: "35"
+    taskid: 7cc51070-069e-4761-80f6-9a4e25f1f6f2
+    type: regular
+    task:
+      id: 7cc51070-069e-4761-80f6-9a4e25f1f6f2
+      version: -1
+      name: Lower alert severity
+      description: commands.local.cmd.set.incident
+      script: Builtin|||setAlert
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "21"
+    scriptarguments:
+      severity:
+        simple: Low
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 1050,
+          "y": 1110
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "36":
+    id: "36"
+    taskid: 89662d70-5716-459a-8743-b4b3e974e784
+    type: condition
+    task:
+      id: 89662d70-5716-459a-8743-b4b3e974e784
+      version: -1
+      name: Should block IP?
+      description: Approve or deny automated remediation.
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "30"
+      "Yes":
+      - "34"
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": 370,
+          "y": 1110
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+view: |-
+  {
+    "linkLabelsPosition": {
+      "28_20_Malicious IP": 0.38,
+      "28_29_#default#": 0.53,
+      "33_20_#default#": 0.49,
+      "36_30_#default#": 0.54,
+      "36_34_Yes": 0.6
+    },
+    "paper": {
+      "dimensions": {
+        "height": 1845,
+        "width": 1340,
+        "x": 90,
+        "y": -110
+      }
+    }
+  }
+inputs:
+- key: InternalRange
+  value: {}
+  required: false
+  description: List of internal IP ranges
+  playbookInputQuery:
```

Comment:
we want to the playbook available for XSIAM only.
Please add the following in the playbook yml file.
marketplaces: ["marketplacev2"] @TalNos

Reply:
done